### PR TITLE
add doc github workflow

### PR DESCRIPTION
**Pull request by galitf**

### CODE REVIEW

File: docs/check_docs.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (tinez) [inline docs/check_docs.sh:1]: Do we need this script at all? Can't we simply run `tox -e docs`?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# oVirt System Tests
+# oVirt System Tests test
```
Comment (tinez) [inline README.md:1]: Some leftover?

**Review by tinez — CHANGES_REQUESTED** [review]

File: .github/workflows/doc.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: check-docs
```
[the remark this answers is not in the document]
Comment (galitf) [reply .github/workflows/doc.yaml]: fixed

File: .github/workflows/doc.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+on: 
+  pull_request:
```
[the remark this answers is not in the document]
Comment (galitf) [reply .github/workflows/doc.yaml]: Changing to this, outputs: You have an error in your yaml syntax on line 2
I will check other options.

File: docs/check_docs.sh (hunk 1 above)
Comment (galitf) [reply docs/check_docs.sh:1]: done

File: .github/workflows/doc.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (galitf) [reply .github/workflows/doc.yaml]: splited to to sections

**Review by tinez — APPROVED** [review]

**Pull request merged**